Detectors/EMCAL/base/include/EMCALBase/Point.h

```diff
@@ -0,0 +1,98 @@
+#ifndef ALICEO2_EMCAL_POINT_H
+#define ALICEO2_EMCAL_POINT_H
+
+#include <FairMCPoint.h>
+
+namespace AliceO2 {
+  namespace EMCAL {
+    
+    /// \class Point
+    /// \brief EMCAL simulation point information
+    class Point : public FairMCPoint {
+    public:
+      
+      /// \brief Default constructor
+      Point();
+      
+      /// \brief Point constructor
+      ///
+      /// Fully defining information of the EMCAL point (position,
+      /// momentum, energy, track, ...)
+      ///
+      /// \param shunt
+      /// \param primary Number of primary particle
+      /// \param trackID Index of the track
+      /// \param parentID ID of the parent primary entering the EMCAL
+      /// \param detID ID of the detector segment
+      /// \param initialEnergy Energy of the primary particle enering the EMCAL
+      /// \param pos Position vector of the point
+      /// \param mom Momentum vector for the particle at the point
+      /// \param tof Time of the hit
+      /// \param length Length of the segment
+      /// \param EventId Event ID
+      Point(Int_t shunt, Int_t primary, Int_t trackID, Int_t parentID, Int_t detID, Int_t initialEnergy, TVector3 pos, TVector3 mom,
+            Double_t tof, Double_t eLoss, UInt_t EventId=0);
```

Comment:
I agree that we should use our class for the hit with less data members: only those which are needed for FairRoot functionality + DetectorId + coordinate/time (in float) should be kept. If detector needs extra info (like momentum), it can be added in detector-specific derived class.
BTW, @mfasDa , the ``Double32_t mInitialEnergy;`` counterpart in AliEMCaliHit is defined but not used. Do we need it here?